src/Test/Utilities/Portable/TargetFrameworkUtil.cs

```diff
@@ -0,0 +1,100 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Text;
+using Microsoft.CodeAnalysis;
+using static TestReferences;
+
+namespace Roslyn.Test.Utilities
+{
+    public enum TargetFramework
+    {
+        /// <summary>
+        /// Explicit pick a target framework that has no references
+        /// </summary>
+        Empty,
+
+        NetStandard20,
+        WinRT,
+
+        /// <summary>
+        /// Eventually this will be deleted and replaced with NetStandard20. Short term this creates the "standard"
+        /// API set across destkop and coreclr 
+        /// </summary>
+        Standard,
+
+        /// <summary>
+        /// This is represents the set of tests which must be mscorlib40 on desktop but full net standard on coreclr.
+        /// </summary>
+        StandardCompat,
+
+        // The flavors of mscorlib we support + extending them with LINQ and dynamic.
+        Mscorlib40,
+        Mscorlib40Extended, 
+        Mscorlib40AndSystemCore,
+        Mscorlib45,
+        Mscorlib45Extended,
+        Mscorlib45AndCSharp,
+        Mscorlib46,
+        Mscorlib46Extended,
+    }
+
+    public static class TargetFrameworkUtil
+    {
+        public static ImmutableArray<MetadataReference> EmptyReferences => ImmutableArray<MetadataReference>.Empty;
```

Comment:
I think this property mostly exists to confuse people, since why would you have it unless it's different from `ImmutableArray.Empty`?

Reply:
I added it out of a need for consistency. But agree it doesn't really add any value here as a property.